.github/workflows/ci.yml

```diff
@@ -1,34 +1,39 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches: [master]
```

Comment:
Why you exclude other branches?

Reply:
Partly a security recommendation, partly a reduction in CI time used. Though it wouldn't be significant for this repo. Can revert if you'd like.